vcd/resource_vcd_vapp_firewall_rules.go

```diff
@@ -361,30 +363,33 @@ var errHelpVappNetworkFirewallRulesImport = fmt.Errorf(`resource id must be spec
 // Example list path (_the_id_string_): list@org-name.vdc-name.vapp-name
 // Note: the separator can be changed using Provider.import_separator or variable VCD_IMPORT_SEPARATOR
 func vappFirewallRuleImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	return vappNetworkRuleImport(d, meta, "vcd_vapp_firewall_rules")
+}
+func vappNetworkRuleImport(d *schema.ResourceData, meta interface{}, resourceType string) ([]*schema.ResourceData, error) {
```

Comment:
Plural more precise?
```suggestion
func vappNetworkRulesImport(d *schema.ResourceData, meta interface{}, resourceType string) ([]*schema.ResourceData, error) {
```

Reply:
changed